mayastor/src/bdev/nexus/nexus_share.rs

```diff
@@ -182,6 +182,24 @@ impl Nexus {
     pub fn get_share_path(&self) -> Option<String> {
         match self.nexus_target {
             Some(NexusTarget::NbdDisk(ref disk)) => Some(disk.get_path()),
+            Some(NexusTarget::NexusIscsiTarget(ref iscsi_target)) => {
+                Some(iscsi_target.as_uri())
+            }
+            _ => None,
+        }
+    }
+
+    /// Return target  /dev/... under which the nexus is shared or None if not
+    /// nbd: path /dev/...
+    /// iscsi: uri
+    pub fn get_target(&self) -> Option<rpc::mayastor::nexus::Target> {
+        match self.nexus_target {
+            Some(NexusTarget::NbdDisk(ref disk)) => {
+                Some(rpc::mayastor::nexus::Target::Nbd(disk.get_path()))
```

Comment:
This does not produce a URI, we should use URI's everywhere to make the code easier to work with. We can either say:
nbd:///dev/nbd0

or 

file:///dev/nd0

I thin the first one is better...

Reply:
Changed to use file:///dev/nbdx